cms/grading/__init__.py

```diff
@@ -177,6 +177,57 @@ def all(self):
                     "code different from 0.")),
 ])
 
+EVALUATION_MESSAGES_AIO = MessageCollection([
+    HumanMessage("success",
+                 N_("Output is correct"),
+                 N_("Your submission ran and gave the correct answer")),
+    HumanMessage("wrong",
+                 N_("Output isn't correct"),
+                 N_("Your submission ran, but gave the wrong answer")),
+    HumanMessage("nooutput",
+                 N_("Evaluation didn't produce file <filename>"),
+                 N_("Your submission ran, but did not write to the "
+                    "correct output file")),
+    HumanMessage("timeout",
+                 N_("Time limit exceeded"),
+                 N_("Your submission ran for too long.")),
+    HumanMessage("div0",
+                 N_("Program crashed after a division by zero error"),
+                 N_("Your program tried to divide by zero. Note that only "
+                    "C, C++ and Pascal programs will get this message, "
+                    "other languages will get return code nonzero instead.")),
+    HumanMessage("segfault",
+                 N_("Program crashed after accessing or requesting "
+                    "invalid memory"),
+                 N_("Your program tried to access or request invalid "
+                    "memory.")),
+    HumanMessage("syscall",
+                 N_("Your submission was stopped because it tried to "
+                    "interact with the operating system in a forbidden way."),
+                 N_("Your submission tried to interact with the operating "
+                    "system in a forbidden way, check the rules regarding "
+                    "what libraries are allowed for your language.")),
+    HumanMessage("signal",
+                 N_("Program crashed for an unknown reason"),
+                 N_("...?")),
```

Comment:
I guess we should replace this with "Please contact the judged via the communication tab", or something? If this ever happens in the AIO we should probably provide direction to the kids.